src/auth-email-whitelist.js

```diff
@@ -15,22 +15,8 @@ const decode = require('jwt-decode');
   return BBAuth
     .getToken()
     .then((token: string) => {
-      const emailDomainWhitelist: string[] = [
-        'blackbaud.ca',
-        'blackbaud.com',
-        'blackbaud.me',
-        'blackbaud.com.au',
-        'blackbaud.co.uk',
-        'blackbaud.au',
-        'microedge.com',
-        'attentive.ly',
-        'everydayhero.com',
-        'smarttuition.com'
-      ];
       const parsedToken = decode(token);
-      const domain = parsedToken.email.split('@')[1];
-
-      if (emailDomainWhitelist.indexOf(domain.toLowerCase()) > -1) {
+      if (parsedToken['1bb.perms'] === 1 || parsedToken['1bb.perms'].indexOf(1) > -1) {
```

Comment:
For my clarification... `1bb.perms` can return a single integer or an array of integers?